### PR TITLE
Fix some problems in reach

### PR DESCRIPTION
**Pull request by kwalcock**
Change $HOME to ${user.home} for compatibility

### CODE REVIEW

**Comment by kwalcock** [comment]
Other serialization types do not yet account for the new Modifications, yet.  Deserialization needs to be handled as well.

**Comment by kwalcock** [comment]
I should probably add at least one test, but otherwise those are the changes that I wanted to make in this round.

File: assembly/src/main/scala/org/clulab/reach/assembly/sieves/AssemblyActions.scala [hunk 1]
```diff
@@ -296,14 +296,8 @@ object AssemblyActions extends LazyLogging {
...
-  def isBlacklisted(words: Seq[String], idx: Int, blacklisted: Seq[String]): Boolean = idx match {
-    case outOfBounds if outOfBounds == -1 || outOfBounds > words.size => false
-    case i =>
-      val w = words(i)
-      val isBlacklisted = blacklisted.contains(w)
-      //logger.info(s"'$w' isBlacklisted? <$isBlacklisted>")
-      isBlacklisted
-  }
+  def isBlacklisted(words: Seq[String], idx: Int, blacklisted: Seq[String]): Boolean =
+    words.isDefinedAt(idx) && blacklisted.contains(words(idx))
```
Comment (myedibleenso) [inline assembly/src/main/scala/org/clulab/reach/assembly/sieves/AssemblyActions.scala:300]: nice.

**Review by myedibleenso — APPROVED** [review]
LGTM.  Thank you, @kwalcock !

**Pull request merged**